codegen.yml

```diff
@@ -59,5 +59,4 @@ generates:
       - ./extractRootFields.js
 hooks:
   afterAllFileWrite:
-    - prettier --write
```

Comment:
I think it's nice to have the .generated files run by prettier, especially since now we're adding them to the repo. I know this adds a few seconds, but I think it's worth it.

Reply:
Yeah, I was questioning it myself because it is kind of nice to have formatted files when you have to look inside generated TS files. I'll revert those commits.